File: src/main/java/net/shadowfacts/underwaterutilities/gui/UUConfigGUI.java
package net.shadowfacts.underwaterutilities.gui;

import net.minecraft.client.gui.GuiScreen;
import net.shadowfacts.underwaterutilities.UUConfig;
import net.shadowfacts.underwaterutilities.UnderwaterUtilities;
import net.shadowfacts.shadowmc.config.GUIConfig;

/**
 * @author shadowfacts
 */
public class UUConfigGUI extends GUIConfig {

	public UUConfigGUI(GuiScreen parent) {
		super(parent, UnderwaterUtilities.modId, UUConfig.config);
	}

}

File: src/main/kotlin/net/shadowfacts/underwaterutilities/gui/UUConfigGUI.kt
package net.shadowfacts.underwaterutilities.gui

import net.minecraft.client.gui.GuiScreen
import net.shadowfacts.shadowmc.config.GUIConfig
import net.shadowfacts.underwaterutilities.MOD_ID
import net.shadowfacts.underwaterutilities.UUConfig

/**
 * @author shadowfacts
 */
class UUConfigGUI(parent: GuiScreen): GUIConfig(parent, MOD_ID, UUConfig.config)
